matchmaker/views/external_api_tests.py

```diff
@@ -291,6 +302,103 @@ def test_mme_match_proxy(self, mock_post_to_slack, mock_email, mock_logger):
         )
         mock_email.assert_not_called()
 
+    @mock.patch('matchmaker.views.external_api.EmailMessage')
+    @mock.patch('matchmaker.views.external_api.safe_post_to_slack')
+    def test_mme_match_proxy_phenotype_only(self, mock_post_to_slack, mock_email):
+        url = '/api/matchmaker/v1/match'
```

Comment:
yes, because this is an external API and we need to maintain this exact URL. If we change something in the URL config and the actual URL you need to access this endpoint changes we would want the unit test to fail, whereas if you use reverse it will automatically update to test the new route